src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -102,14 +102,17 @@ private FolderComposite FindInputFiles(string path, List<string> filesToExclude,
             }
             foreach (var file in _fileSystem.Directory.GetFiles(folderComposite.FullPath, "*.cs", SearchOption.TopDirectoryOnly))
             {
-                var fileName = Path.GetFileName(file);
-                folderComposite.Add(new FileLeaf()
+                if (!file.EndsWith(".xaml.cs"))
```

Comment:
I would like some comment on why we are excluding xaml.cs files here so that we can find it back later